### ISSUE TITLE
Yesterday's update on the US Traval Ban

### ISSUE THREAD

**Issue by jglovier** [aymannadeem/yourhaq#42]
Yesterday [the Supreme Court ruled](https://www.nytimes.com/2017/12/04/us/politics/trump-travel-ban-supreme-court.html) on the travel ban such that it is able to be implemented while it undergoes further legal challenges. This seems like an important update for YourHAQ visitors.

@aymannadeem what do think is the smallest update to the current site to best share recent updates like this? A News section seems like it could be appropriate?

I could try to get a PR going to add that in the next couple weeks.

In lieu of that for the immediate, is there any existing place for recent news updates to go?

**Comment by aymannadeem** [comment]
Thanks for the sharing @jglovier! 

This is definitely valuable information. Currently, there is no place for news updates. 

My hesitation with such a section is that the news cycle [moves so fast](https://www.wired.com/story/historians-see-terrifying-echoes-in-trumps-anti-muslim-tweets/?mbid=social_twitter_onsiteshare) that it may become outdated and unmaintainable very quickly. Unless there was a concerted effort from a dedicated human to read and summarize the minimum valuable information from each legislative update, it would become stale fast.

One idea I have is to have a visualization showing nations currently affected by the travel ban. It would be cool to have something where, if you hovered over a nation, it would have a small blurb on what the travel ban means for that country (ex., hovering over Iran vs. Yemen, etc.) 

What do you think?

**Comment by jglovier** [comment]
> My hesitation with such a section is that the news cycle moves so fast that it may become outdated and unmaintainable very quickly. Unless there was a concerted effort from a dedicated human to read and summarize the minimum valuable information from each legislative update, it would become stale fast.

Yeah, that makes sense. My thinking was that if the community around this project grew and was active enough, the community could decide what information belongs there by submitting PRs and vetting them, etc. But until that point the data def could become stale pretty fast.

>One idea I have is to have a visualization showing nations currently affected by the travel ban. It would be cool to have something where, if you hovered over a nation, it would have a small blurb on what the travel ban means for that country (ex., hovering over Iran vs. Yemen, etc.)

Yeah, that's a great idea. I really like that. How frequently do you think blurbs like that would need updated?

**Comment by aymannadeem** [comment]
> How frequently do you think blurbs like that would need updated?

As frequently as the countries and the immigration constraints around them change? ¯\_(ツ)_/¯
